test/html/cases/recent.mli

```diff
@@ -26,3 +26,11 @@ type polymorphic_variant = [
 ]
 
 type nonrec nonrec_ = int
+
+
+(* Conjunctive types: dune compilation scheme exposes a bug in old
+   versions of the compiler *)
```

Comment:
What does this refer to?

Reply:
I was thinking of https://github.com/ocaml/ocaml/issues/7629 , but there were other issues with conjunctive types in older version of ocaml : https://github.com/ocaml/ocaml/issues/7496 for instance. The short explanation is that conjunctive types are intended as type-checking artefacts and using them directly can make compilation break in unexpected way.